mobile/src/Student/screens/Game/BasicGamePlay/index.js

```diff
@@ -73,47 +86,90 @@ const GamePreview = ({ navigation, route, gameSession, team, teamMember }) => {
-        global.apiClient
-            .addTeamAnswer(
-                teamMember.id,
-                question.id,
-                answer.text,
-                answer.isSelected
-            )
-            .then((teamAnswer) => {
-                if (teamAnswer == null) {
-                    console.error("Failed to create team.")
-                    return
-                }
-                console.log(teamAnswer)
-            })
-            .catch((error) => {
-                console.error(error.message)
-            })
-        if (!gameSession.isAdvancedMode) {
-            navigateToNextScreen()
-        }
+        Alert.alert(
+            "Are you sure?",
+            "You will not be able to change your answer",
+            [
+                {
+                    text: "Cancel",
+                    style: "cancel",
+                },
+                {
+                    text: "OK",
+                    onPress: () => {
+                        global.apiClient
+                            .addTeamAnswer(
+                                teamMember.id,
+                                question.id,
+                                answer.text,
+                                answer.isSelected,
+                                answer.isChosen
+                            )
+                            .then((teamAnswer) => {
+                                if (teamAnswer == null) {
+                                    console.error(
+                                        "Failed to create team Answer."
+                                    )
+                                    return
+                                }
+                                teamAnswer.isChosen = true
+                                console.log("this is team answer", teamAnswer)
+                            })
+                            // .then(() => {
+                            //     if (!gameSession?.isAdvanced) {
+                            //         navigateToNextScreen()
+                            //     }
+                            // })
+                            .catch((error) => {
+                                console.error(error.message)
+                            })
+                    },
+                },
+            ]
+        )
     }
 
     const answersParsed = JSON.parse(question.choices)
 
     const answerChoices = answersParsed.map((choice) => {
-        return choice.text
+        return {
+            id: uuid.v4(),
+            text: choice.text,
+            isCorrectAnswer: choice.isAnswer,
+            isSelected: false,
+        }
     })
 
+    //if isCorrectAnswer is true, add 10 points to the team's score
+    const addPoints = (answer) => {
+        if (answer.isCorrectAnswer) {
+            team.score += 10
+        }
+    }
+
+    const handleAnswerResult = (answer) => {
+        setSelectedAnswer(answer)
+        submitAnswer(answer)
+        addPoints(answer)
```

Comment:
Might wanna do the points when `submitAnswer` is successful.

Reply:
Yea this scoring needs work in general! Will get back around to that one, thank you